packages/ui-extensions/src/surfaces/admin/components/Modal/Modal.doc.ts

```diff
@@ -0,0 +1,39 @@
+import {AdminReferenceEntityTemplateSchema} from '../../docs-types';
```

Comment:
Let's rename this to AB.doc.ts or whatever the naming convention is until we know for sure if this is going to ui-extensions